editions/tw5.com/tiddlers/filters/Filter Operators.tid

```diff
@@ -18,13 +18,20 @@ type: text/vnd.tiddlywiki
 <tr class="doc-table-subheading"><th colspan="5" align="center">$_$</th></tr>
 \end
 
-A <<.def "filter operator">> is a predefined keyword attached to an individual [[step|Filter Step]] of a [[filter|Filters]]. It defines the particular action of that step.
+A <<.def "filter operator">> is a predefined keyword attached to an individual step of a [[filter|Filters]]. It defines the particular action of that step.
 
-''Important:'' Each first [[step|Filter Step]] of a [[filter run|Filter Run]] not given any input titles receives the output of <$link to="all Operator">[all[tiddlers]]</$link> as its input.
+''Important:'' Each first [[filter step|Filter Step]] of a [[filter run|Filter Run]] not given any input titles receives the output of <$link to="all Operator">[all[tiddlers]]</$link> as its input.
 
-The following table lists all core operators, the most common ones marked ✓. The `!` column indicates whether an operator allows ''negation'' using the <$link to="Filter Step"><code>!</code> prefix</$link>. For specifics as to each operator's negated output please refer to its documentation. 
 
-Most steps process the [[selection of titles|Title Selection]] that are supplied as their input, but a few [[construct an entirely new selection|Selection Constructors]] instead, as indicated by the last column. A `C?` indicates it might construct a new selection, depending on usage. For specifics as to each operator's selection creation please refer to its documentation. 
+''Table legend:''
+
+* ✓ .. Used to mark the most common ones
```

Comment:
Double period doesn't have a clear meaning in English, but perhaps you were aiming for an ellipsis?

An ellipsis would be incorrect here. The meaning of an ellipsis is to mark a pause or something that has been left out. The former meaning is never appropriate in formal documentation like this. The latter meaning is needed in some situations such as edited quotations.

Reply:
I did try a table which will look like this. I think this looks better

![image](https://user-images.githubusercontent.com/374655/226197830-0010a2e8-ef5f-44c3-a9ba-35e30d3f56b3.png)